Documentation/advanced-configuration.md

```diff
@@ -433,3 +434,34 @@ other OSDs holding replica data are unavailable:
 ```bash
 ceph osd primary-affinity osd.0 0
 ```
+
+## Phantom OSD Removal
+
+If you have OSDs in which are not showing any disks, you can remove those "Phantom OSDs" by following the instructions below.
+To check for "Phantom OSDs", you can run:
+```bash
+ceph osd tree
+```
+An example output looks like this:
+```
+ID  CLASS WEIGHT  TYPE NAME STATUS REWEIGHT PRI-AFF
+ -1       57.38062 root default
+-13        7.17258     host node1.example.com
+  2   hdd  3.61859         osd.2                up  1.00000 1.00000
+ -7              0     host node2.example.com   down    0    1.00000
+```
+The host `node2.example.com` in the output has no disks, so it is most likely a "Phantom OSD".
+
+Now to remove it, use the ID in the first column of the output and replace `<ID>` with it.
```

Comment:
In this case, the `<ID>` would be `-7`, correct? You might mention that here in the example.

Reply:
@travisn Done.